### PR TITLE
stickers: allow kanging video stickers

### PR DESCRIPTION
**Pull request by itsLuuke**

### CODE REVIEW

File: tg_bot/modules/stickers.py [hunk 1]
```diff
@@ -289,12 +311,20 @@ def kang(update: Update, context: CallbackContext):  # sourcery no-metrics
...
                 tgs_sticker=sticker_data if is_animated else None,
-                png_sticker=sticker_data if not is_animated else None,
+                webm_sticker=sticker_data if is_video else None,
+                png_sticker=sticker_data if not is_animated and not is_video else None,
             )
         elif e.message == "Stickers_too_much":
             msg.reply_text("Max packsize reached. Press F to pay respecc.")
         elif e.message == "Invalid sticker emojis":
             msg.reply_text("I can't kang with that emoji!")
+        elif e.message == "Sticker_video_nowebm":
```
Comment (Dank-del) [inline tg_bot/modules/stickers.py:321]: oo ic there are quite an amount of error types
maybe ill make this a string enum later on

File: tg_bot/modules/stickers.py [hunk 2]
```diff
@@ -138,17 +145,26 @@ def kang(update: Update, context: CallbackContext):  # sourcery no-metrics
...
+            file_id = rep.sticker.file_id
             # also grab the emoji if the user wishes
             if not args:
-                sticker_emoji = msg.reply_to_message.sticker.emoji
-        elif msg.reply_to_message.photo:
-            file_id = msg.reply_to_message.photo[-1].file_id
-        elif msg.reply_to_message.document:
-            file_id = msg.reply_to_message.document.file_id
+                sticker_emoji = rep.sticker.emoji
+        elif rep.photo:
```
Comment (Dank-del) [inline tg_bot/modules/stickers.py:156]: aaa i wish server had 3.10 python then we could use switch case here

File: tg_bot/modules/stickers.py [hunk 3]
```diff
@@ -360,6 +392,14 @@ def makepack_internal(
             == 'Internal Server Error: created sticker set not found (500)'
         ):
             success = True
+        elif e.message == 'Sticker_video_nowebm':
+            msg.reply_text(
+                "This media format isn't supported, I need it in a webm format, "
+                "[see this guide](https://core.telegram.org/stickers/webm-vp9-encoding).",
```
Comment (Dank-del) [inline tg_bot/modules/stickers.py:398]: o til they had a guide for this

File: tg_bot/modules/stickers.py (hunk 2 above)
Comment (itsLuuke) [reply tg_bot/modules/stickers.py:156]: Yep thats why i kept this syntax

**Pull request merged**